cwatch.php

```diff
@@ -908,49 +908,53 @@ private function getMalwareTab($template, $service, $post)
 
         // Load the helpers required for this view
         Loader::loadHelpers($this, ['Form', 'Html']);
+        Loader::loadComponents($this, ['Security']);
 
         // Get cWatch API
         $api = $this->getApi();
         $service_fields = $this->serviceFieldsToObject($service->fields);
 
         if (isset($post['test_ftp'])) {
-            $use_ftps = isset($post['ftps']);
-            $error_message = Language::_('CWatch.!error.ftp' . ($use_ftps ? 's' : '') . '_test', true);
+            $use_sftp = isset($post['sftp']);
```

Comment:
This should probably be Html::ifSet

Reply:
It is a checkbox, I don't care about the value, only that it is set